domains/caiden.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "Caiden-P",
+        "email": "iididhejejdj@gmail.com"
+    },
+    "record": {
+        "A": ["217.174.245.249"],
+        "MX": ["hosts.is-a.dev"],
+        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
```

Comment:
Hosting by Is-a.dev was shutdown 1 month ago

Reply:
oh lmao thats why, i fixed it but i want to put my discord as a contact and im unsure should it be my username or my userid and if userid should it be in string format or int?

Reply:
or does it have to be both email and discord? i just removed it cuz i dont check that email

Reply:
> oh lmao thats why, i fixed it but i want to put my discord as a contact and im unsure should it be my username or my userid and if userid should it be in string format or int?

better the ID, and as string

Reply:
> or does it have to be both email and discord? i just removed it cuz i dont check that email

if you give any other social, you can keep the mail field as an empty string (otherwise the CI fails)

Reply:
ok ill updated everything thanks for the help

Reply:
should be alright now i think